src/patternfly/components/Table/docs/table-compact.md

```diff
@@ -12,3 +12,4 @@
 | Class | Applied To | Outcome |
 | -- | -- | -- |
 | `.pf-m-compact` | `.pf-c-table` | Modifies for a compact table. |
+| `.pf-m-border-rows` | `.pf-c-table` | Modifies border rows to have a border. |
```

Comment:
I think you should update the description to be more specific -  does this modifier give all rows  a border or can you modify individual rows?
andddd since all other tables have borders can it be applied to them? or should this just apply to `.pf-c-table .pf-c-compact`

Reply:
since it only applies to `.pf-c-table`, I think it's implied you just get borders on the whole table, not on individual rows. Updated the docs so it only applies to `.pf-c-table.pf-m-compact`